nbconvert/exporters/webpdf.py

```diff
@@ -71,7 +67,7 @@ def _template_name_default(self):
 
     def _check_launch_reqs(self):
         try:
-            from pyppeteer import launch
+            from pyppeteer import launch  # type: ignore
```

Comment:
I like having the type ignore codes when ignoring types, but if it is not standard to do here then ignore my comment.